gnomad_hail/resources/grch38/reference_data.py

```diff
@@ -11,14 +11,32 @@
     path="gs://gnomad-public/resources/grch38/purcell5k.ht",
     import_sources={
         "source_path": "gs://gnomad-public/intervals/purcell5k.interval_list",
-    },
+    }
```

Comment:
I think @mike-w-wilson was using [Black](https://black.readthedocs.io/en/stable/) and it put these commas here.

It would be nice if we agreed on a style guide (or better yet, all ran Black 😁) to avoid stylistic churn like this.

Reply:
We've never left stray commas like this anywhere else... I'm not necessarily opposed to it but it should be applied all over if we do

Reply:
Yup! I did run Black on this. Definitely all for agreeing and adhering to a style guide - not sure if this fits into our recent standardization meetings but maybe we could approach it there.

Reply:
Sounds like a good idea. For now, I'll merge this since anyways the entire lib will need to be formatted once we agree on a format.

Reply:
Agreed that consistency throughout is preferable. A style guide would be one way to achieve that, but usually ends up with lots of nitpick comments in PR reviews. Alternatively, a formatter applies the style guide automatically.

Also, trailing commas are useful to reduce noise in diffs.
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas

Reply:
> I'll merge this since anyways the entire lib will need to be formatted once we agree on a format

👍 Certainly did not mean for that comment to be taken as blocking.

Reply:
Having used formatters in JavaScript (and Black a little bit), I can say this is a game-changer! It's disconcerting at first, but you'll progress to a point where you can't live without it 😂.

Funny thread from JS community on this subject: https://twitter.com/dan_abramov/status/1161998154138603525?lang=en